docs/release-notes/october-2024.md

```diff
@@ -0,0 +1,130 @@
+---
+slug: october-2024
+title: October 2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for October 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+- Added a **Scan details** page and drawer for all completed scans. Use this page to troubleshoot or view information about individual scans. 
+![Scan details drawer with the permalink icon indicated in a box.](/img/scan-details-permalink.png)
+_**Figure**. Scan details drawer with the permalink icon indicated in a box._
+- The **Dashboard** now provides a **Teams** filter, enabling you to create views based on a selection of [Teams](/deployment/teams#teams-beta) you are a part of. Click **Dashboard > Filters** to access the filter.
+  - By default, the Dashboard now displays findings from teams you are a part of. Your finding count may differ from your colleagues based on your Teams.
+- Added a Jira API endpoint to create Jira tickets, either by passing a list of `issue_ids` or filter query parameters to select findings. Refer to the [<i class="fas fa-external-link fa-xs"></i> Jira API documentation](https://semgrep.dev/api/v1/docs/#tag/TicketingService/operation/semgrep_app.core_exp.notifications.ticketing.handlers.openapi_create_tickets).
+- Semgrep now supports [Move on Sui](https://docs.sui.io/concepts/sui-move-concepts), thanks to the contributions of the Sui team.
+
+### Changed
+
+- Various UI improvements to the **Settings > SCM** tab.
+![Old SCM card](/img/old-scm-card.png) 
+![Updated SCM card](/img/new-scm-card.png)
+_**Figure**. Previous and current SCM card UI._
+- **Semgrep Managed Scans**: scans now follow fail open behavior, consistent with how Semgrep in CI behaves. 
+- The **Projects** page's **See findings** button is now a drop-down box, enabling you to select which product you want to view findings for.
+
+### Fixed
+
+- When a scan runs into an exception, Semgrep AppSec Platform is appropriately notified about the failure. Previously, within the AppSec Platform UI, it would appear to the user that the scan is still in progress.
```

Comment:
```suggestion
- When a scan runs into an exception, Semgrep AppSec Platform displays information about the failure. Previously, within the AppSec Platform UI, it would appear to the user that the scan is still in progress.
```